tests/neptune/test_experiment.py

```diff
@@ -150,6 +150,46 @@ def build_call(tags_list):
             build_call(['tag1', 'tag2', 'tag3'])
         ])
 
+    def test_delete_artifact(self):
+        # given
+        backend = mock.MagicMock()
+        project = a_project()
+        experiment = Experiment(
+            backend,
+            project,
+            an_experiment_id(),
+            a_uuid_string()
+        )
+
+        # and
+        def build_call(paths):
+            return call(
+                project=project,
+                paths=paths,
+                recursive=True
+            )
+
+        # when
+        experiment.delete_artifacts('/an_abs_path_in_exp_output')
+        experiment.delete_artifacts('/../an_abs_path_in_exp')
+        experiment.delete_artifacts('/../../an_abs_path_in_prj')
+        experiment.delete_artifacts('a_path_in_exp_output')
+        experiment.delete_artifacts('../a_path_in_exp')
+        experiment.delete_artifacts('../../a_path_in_prj')
```

Comment:
Why all those `../` paths do not throw exceptions?

Reply:
because we prepend user paths with shortid/output client-side before sending to server